### ISSUE TITLE
Training data consists of oriented bounding boxes rather than horizontal ones

### ISSUE THREAD

**Issue by vsd550** [ruotianluo/pytorch-faster-rcnn#93]
I want to perform classification and detection only on my dataset of satellite images using Faster Rcnn. The problem is that there are very **dense objects** in the image and have _oriented bounding boxes with all the 4 coordinates specified_, while the faster rcnn takes horizantal bounding boxes with 2 coordinates only(Xmin,Ymin,Xmax,Ymax) while training.
SO basically we need to regress over 4 coordinates rather than 2.
Please suggest how can I do it or refer me to some similar projects.
Thanks in advance

**Comment by mubastan** [comment]
Standard object detectors (Faster R-CNN, SSD, YOLO,...) all use upright bounding boxes.
If the bounding boxes are not upright and they are very dense, it may be better to use a segmentation algorithm (e.g., Mask R-CNN), as you probably have segmentation masks (roughly).

**Comment by vsd550** [comment]
@Shamsa279 Unfortunately not yet. I am on it. Please share if u find something.
Thanks

**Comment by priya55612** [comment]
@Shamsa279  and @vsd550  Did you guys find anything on this?

**Comment by priya55612** [comment]
My bounding boxes have center (x,y), height, width, and angle
